src/components/organisms/watchPartyContent/WatchPartyContent.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Card as AntdCard } from 'antd';
+import { Button, Col, Input, Image, Row, Typography } from 'antd';
+
+import Context from '../../../context';
+import styles from './watchPartyContent.module.scss';
+import MovieImg from '../../../assets/images/second.png';
+import background from '../../../assets/images/second.png';
+
+const WatchPartyContent = ({ movieName }) => {
+	const isMobile = useContext(Context.DeviceContext);
+
+	return (
+		<>
+			<div className={styles.main}>
+				<Col flex={7}>
+					<div
+						style={{
+							backgroundImage: `url(${background})`,
```

Comment:
please use css module i.e. write a css class and add the same under className property; you could write 2 classes - one for mobile & another for desktop and apply them conditionally